src/stirling/core/data_tables.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "src/stirling/core/data_table.h"
+#include "src/stirling/core/types.h"
+
+namespace px {
+namespace stirling {
+
+class DataTables {
```

Comment:
Will we be using this class outside of any test scenarios? It seems like this class's responsibility is still test focused, so it seems slightly out of place to be in `src/stirling/core`.

Reply:
@etep did you see my previous comment about this?

Reply:
We talked about this offline. I missed that this was used in the UnitConnector 😅